System.Device.I2s/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="nanoFramework.CoreLibrary" version="1.12.0" targetFramework="netnanoframework10" />
-  <package id="Nerdbank.GitVersioning" version="3.5.107" developmentDependency="true" targetFramework="netnanoframework10" />
+  <package id="nanoFramework.Runtime.Events" version="1.10.0" targetFramework="netnano1.0" />
```

Comment:
Are you sure Events are required? 🤔

Reply:
I don't think so. As the work has been based on the I2C package, I assume that dependency comes from there. I dropped it and updated the nuspec files.